lib/dns/src/opstrace.ts

```diff
@@ -359,6 +359,11 @@ export class DNSClient {
             die("DNS setup failed with a permanent error (403 HTTP response).");
           }
 
+          // exit if we're trying to use a cluster name that is already taken
+          if (e.response?.statusCode === 409 && method === "POST") {
+            die("DNS setup failed with a permanent error, please choose a different cluster name");
```

Comment:
let's maybe indicate the conflict/"name already in use" error in the error message. Of course the solution is good to have as in "choose a different one", but exposing the actual problem is also good UX in this case. :)